src/net/tcp.rs

```diff
@@ -333,7 +333,7 @@ impl TcpStream {
     /// "would block" error is returned. This operation does not block.
     ///
     /// On Unix this corresponds to the `writev` syscall.
-    pub fn write_bufs(&self, bufs: &[&IoVec]) -> io::Result<usize> {
+    pub fn write_bufs(&mut self, bufs: &[&IoVec]) -> io::Result<usize> {
```

Comment:
We are going to want to keep this as `&self`.

Reply:
This method is gone on master and is replaced by the `io::Write` trait.